templates/overrides/navigation/block--system-branding-block.html.twig

```diff
@@ -20,7 +20,8 @@
     </a>
   {% endif %}
   {% if site_name %}
-    <p class="site-name"><a class="text-white text-decoration-none align-bottom" href="{{ path('<front>') }}" rel="home">{{ site_name }}</a></p>
+  {% set text_color_class = bcl_header_style =='light' ? 'text-dark': 'text-white' %}
```

Comment:
Since we start adding classes dynamically it starts making sense to create a attributes object and use addClass method, add by default "ext-decoration-none align-bottom" and put the if after adding correct text- class.
Add also href, rel.